owlbot.py

```diff
@@ -15,14 +15,21 @@
 import os
 import re
 import subprocess
+from logging import Logger
 from pathlib import Path
 
 from synthtool import shell
 from synthtool.log import logger
 
+logger: Logger = logger
+
 _TOOLS_DIRECTORY = "/synthtool"
 _EXCLUDED_DIRS = [r"node_modules", r"^\."]
-_TYPELESS_EXPRESSION = "s/export {};$/\\n/"
+_TRIM_EXPRESSIONS = [
```

Comment:
I'd create a tracking ticket for each of these replacements required, assigned to @feywind as a feature request in googleapis/google-clou-node.

@feywind, I know this is probably a secondary priority currently, so don't take this as a request you need to act on immediately.

Reply:
Filed https://github.com/googleapis/google-cloud-node/issues/4057